src/ScheduleService.Models.Dto/Models/GroupUserInfo.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ScheduleService.Models.Dto.Models;
+
+public class GroupUserInfo
+{
+  public Guid GroupId { get; set; }
+
+  public Guid UserId { get; set; }
+}
```

Comment:
let it be for now but keep in mind it must be reworked later(to have real info from user service instead of just id)